common/thorhelper/thorsoapcall.cpp

```diff
@@ -507,6 +509,27 @@ class BlackLister : public CInterface, implements IThreadFactory
     }
 } *blacklist;
 
+static IPersistentHandler* persistentHandler = nullptr;
+static Mutex persistentMutex;
+static bool initDone = false;
```

Comment:
Perhaps a more specific variable name?

Reply:
OK I can change it to a more specific name like persistentInitDone